### PR TITLE
Updated install instructions for Kubeless

### PR DESCRIPTION
**Pull request by jbianquetti-nami**
Installation method has changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Make sure you have a kubernetes endpoint running (e.g minikube) and kubeless ins
 $ minikube version
 $ kubectl version
 $ brew install kubeless/tap/kubeless
```
Comment (sebgoa) [inline README.md]: actually we need to improve this...no everyone is on OSX and/or uses brew

**Comment by sebgoa** [comment]
ok let's merge this, because this is closer to what we have., now that we removed the 'kubeless install'

**Comment by sebgoa** [comment]
@andresmgot can you merge, looks like I don't have karma :(

**Comment by andresmgot** [comment]
Sure

**Pull request merged**